### PR TITLE
pylint fixes

### PR DESCRIPTION
**Pull request by ankurkhemani**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Review by ehotinger — APPROVED** [review]

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py [hunk 1]
```diff
@@ -96,7 +94,7 @@ def _stream_logs(byte_size,
         metadata = props.metadata
         available = props.properties.content_length
         last_modified = props.properties.last_modified
-    except:
+    except (AttributeError, KeyError, TypeError):
```
Comment (djyou) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py]: `get_blob_properties` might throw `CloudError`? Do you need `KeyError` here?

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py [hunk 2]
```diff
@@ -182,18 +180,18 @@ def _stream_logs(byte_size,
             num_fails += 1
 
             logger.debug(
-                "Failed to find new content '{}' times in a row".format(num_fails))
+                "Failed to find new content '%d' times in a row", num_fails)
```
Comment (djyou) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py]: Do you think we can always use `%s`? What's the benefit that `%d` provides?

**Review by djyou — APPROVED** [review]
Just some minor comments.

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py (hunk 2 above)
Comment (ankurkhemani) [reply src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py]: Integers need %d and strings need %s?

Comment (djyou) [reply src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py]: I think unless you want to format the int specifically, string should just be fine. The thing is then you also need to make sure, for example, `num_fails` is indeed an int but we actually don't care even if it's a string int.

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py (hunk 1 above)
Comment (ankurkhemani) [reply src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py]: Fixed

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py (hunk 2 above)
Comment (ankurkhemani) [reply src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py]: Replaced %d with %s

**Pull request merged**